extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisUtils.scala

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.spark.streaming.kinesis
 
+import com.amazonaws.auth.AWSCredentials
```

Comment:
my initial attempt at adding this support introduced a BasicAWSCredentialsProvider as follows:

https://github.com/cfregly/spark/blob/kinesis-dbc/extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/BasicAWSCredentialsProvider.scala

highlights:
1) takes AccessKey and SecretKey as Strings
2) follows the same hierarchy as the rest of the AWSCredentialsProviders (including not being Serializable)